packages/preview/hsh-thesis/1.0.0/src/lib.typ

```diff
@@ -0,0 +1,269 @@
+#import "@preview/hydra:0.3.0": hydra
+#import "@preview/codly:1.0.0": *
+
+#let smallLine = line(length: 100%, stroke: 0.045em)
+
+#let getCurrentHeadingHydra(loc, topLevel: false, topMargin) = {
+    if(topLevel){
+      return hydra(1, top-margin:topMargin)
+    }
+
+    return hydra(2, top-margin:topMargin)
+}
+
+#show par: it => [#it <meta:content>]
+
+#let project(
+  title: "",
+  subtitle: "",
+  author: "",
+  author_email: "",
```

Comment:
The typical and recommended Typst naming convention is kebab-case, i.e. `author-email`.